aspnetcore/blazor/fundamentals/additional-scenarios.md

```diff
@@ -218,6 +218,36 @@ Blazor.defaultReconnectionHandler._reconnectionDisplay =
 
 The placeholder `{ELEMENT ID}` is the ID of the HTML element to display.
 
+::: moniker range=">= aspnetcore-5.0"
+
+## Influence HTML `<head>` tag elements
+
+*This section applies to Blazor WebAssembly and Blazor Server.*
+
+When rendered, the `Title`, `Link`, and `Meta` components add or update data in the HTML `<head>` tag elements:
+
+```razor
+@using Microsoft.AspNetCore.Components.Web.Extensions
+
+<Title Value="{TITLE}" />
+<Link rel="stylesheet" href="{FILE NAME}" />
+<Meta name="description" content="{DESCRIPTION}" />
+```
+
+In the preceding example, placeholders for `{TITLE}`, `{FILE NAME}`, and `{DESCRIPTION}` are sting values, Razor variables, or Razor expressions.
+
+The following characteristics apply:
+
+* Server-side prerendering is supported.
+* The `Value` parameter is only valid for the `Title` component.
+* All provided attributes are reflected in the rendered HTML tags.
+* For multiple `Title` components, the title of the page reflects the `Value` of the last `Title` component in the Razor markup.
```

Comment:
This is true for most cases, but if any `Title` is re-rendered, it will change the page title regardless of its location in the markup (e.g. if a `Value` parameter changes, that change will be applied to the site).

```suggestion
* For multiple `Title` components, the title of the page reflects the `Value` of the last `Title` component to be rendered.
```

Maybe there's a better way to word this than what I've suggested?